src/components/Dashboard/Dashboard.module.scss

```diff
@@ -119,15 +126,65 @@
 .btn {
   width: 45%;
   padding: 0.5rem;
-  color: $severely-burnt-toast;
-  background-color: $translucent-silk;
-  border: none;
+  color: $translucent-silk;
+  background-color: $hornblende-green;
+  border: 1px solid $translucent-silk;
   border-radius: 10px;
   font-size: 1rem;
   font-weight: 300;
   cursor: pointer;
 }
 
 .btn:hover {
-  background-color: $spruce-stone;
+  background-color: $translucent-silk;
+  color: $severely-burnt-toast;
+}
+
+@media only screen and (min-width: 992px) {
+  .box {
+    width: 80%;
+  }
+
+  .title {
+    font-size: 2rem;
+  }
+
+  .activities {
+    flex-direction: row;
+    flex-wrap: wrap;
+    row-gap: 1.8rem;
+    column-gap: 5rem;
+    width: 80%;
+    margin: 2.2rem 0;
+  }
+
+  .activity {
+    width: 43%;
+    height: 4rem;
+    padding: 0 1rem;
+    font-size: 1.4rem;
+  }
+
+  .btnDone {
+    width: 2.5rem;
+    height: 2.5rem;
+  }
+
+  .revealForm {
+    width: 3.5rem;
+    height: 3.5rem;
+    font-size: 3rem;
+  }
+
+  .form {
+    gap: 1.5rem;
+    width: 80%;
+    padding: 2rem;
+    margin: 2rem 0 3rem 0;
+    font-size: 1.4rem;
+  }
+
+  .btn {
```

Comment:
There is no need to use abbreviations. They are just making stuff harder to read and don't not "save" any space. Try to avoid them.

The only abbreviations that make sense are the ones where the abbreviation became the globally known name of the thing. For example "URL"